build.psm1

```diff
@@ -1320,20 +1325,26 @@ function Publish-NuGetFeed
 {
     param(
         [string]$OutputPath = "$PSScriptRoot/nuget-artifacts",
-        [Parameter(Mandatory=$true)]
-        [string]$VersionSuffix
+        [Parameter(ParameterSetName = "ReleaseTag")]
+        [ValidatePattern("^v\d+\.\d+\.\d+(-\w+\.\d+)?$")]
+        [ValidateNotNullOrEmpty()]
+        [string]$ReleaseTag
```

Comment:
I see there we call `Start-PSPackage` - we need fix it too. And in `Start-PSPackage` we have another code path for nuget packages - I wonder why we have two code path to generate nuget packages? Because `Publish-NuGetFeed` was not moved in Packaging.psm1 I thought it is not used on CI but it is used.